docs/conf.py

```diff
@@ -0,0 +1,21 @@
+# Sphinx docs config
+
+project = "sphinx-contribs"
+copyright = "2024, pyOpenSci Community"
+author = "pyOpenSci Community"
+release = "0.1"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = ["sphinx-contribs"]
```

Comment:
Now, even if i install this extension, sphinx can't "see" or find it when you run sphinx-build